subjects/devops/in-back-ground/README.md

```diff
@@ -0,0 +1,52 @@
+## in-back-ground
+
+### Instructions
+
+Create a script `in-back-ground.sh` which will execute a job in the background that will do the following:
+
+- Run the command `cat` on the file "facts" which will read the contents of the file and print it to `stdout`.
+- The output of the cat command will be piped to the `grep` command, which will search for the string `"moon"` in the file.
+- If the `grep` command succeeds ( if it finds the string "moon"), the echo command will run and print `"The moon fact was found!"` to the `output.log` file.
+- If the command finds the string "moon", it will print the matching line to the `stdout`.
+- Redirect the `stdout` to a new file `output.log`.
+- Redirect `stderr` to the same place as `stdout` (in this case, the output.log file).
```

Comment:
I cannot see when this point is being checked or how to get a bash error message in the output.log file - should we just remove this point?

Reply:
removed